relay-kafka/src/producer/mod.rs

```diff
@@ -332,3 +329,67 @@ impl Producer {
             })
     }
 }
+
+struct Debounced {
```

Comment:
Needed now because we need a `Producer: Send` and the atomic impl is a bit more elaborate